OneSignalSDK/build.gradle

```diff
@@ -1,36 +1,45 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
-    ext.kotlin_version = '1.3.72'
+
+    ext {
+        buildVersions = [
+                compileSdkVersion: 28,
+                targetSdkVersion: 28
+        ]
+        googleServicesVersion = '4.3.2'
+        gradleVersion = '3.6.2'
```

Comment:
This is being used for `com.android.tools.build:gradle` so we should call it `androidGradlePluginVersion`.